Configuration/StandardSequences/python/Validation_cff.py

```diff
@@ -59,20 +62,40 @@
 
 prevalidation_preprod = cms.Sequence( preprodPrevalidation )
 
+validation_preprodNoHLT = cms.Sequence(
+                            genvalid_all
+                            +trackingTruthValid
+                            +tracksValidation
+                            +METRelValSequence
+                            +recoMuonValidation
+                            +muIsoVal_seq
+                            +muonIdValDQMSeq
+                          )
+
 validation_preprod = cms.Sequence(
-                          genvalid_all
-                          +trackingTruthValid
-                          +tracksValidation
-                          +METRelValSequence
-                          +recoMuonValidation
-                          +muIsoVal_seq
-                          +muonIdValDQMSeq
+                          validation_preprodNoHLT
                           +hltvalidation_preprod
                           )
 
+_validation_preprod_fastsim = validation_preprod.copy()
+for _entry in [hltvalidation_preprod]:
+    _validation_preprod_fastsim.remove(_entry)
+from Configuration.Eras.Modifier_fastSim_cff import fastSim
+fastSim.toReplaceWith(validation_preprod,_validation_preprod_fastsim)
```

Comment:
same here - if you use the `fastSim` era to modify `validation_preprod`, there is no need to define a separate `validation_preprodNoHLT` above, right ?